doc/install/advanced.rst

```diff
@@ -191,6 +191,42 @@ or by doing
 :func:`mne.viz.set_3d_options(antialias=False) <mne.viz.set_3d_options>` within
 a given Python session.
 
+Another issue that may come up is that the MESA software itself may be out of date
+in certain operating systems, for example CentOS. This may lead to incomplete
+rendering of some 3D plots. A solution is to build the latest version
+locally following `these instructions <https://xorg-team.pages.debian.net/xorg/howto/build-mesa.html#_preparing_mesa_sources>`_.
```

Comment:
I would also (first) link to the instructions that I posted in a comment on GitHub. The 18.x series has a simpler workflow that's easier to satisfy the requirements for than 19+, hence why I built that rather than a 19 or 20 variant

Reply:
okay I added the link. Hope it's the right comment?